docs/hooks.md

```diff
@@ -0,0 +1,259 @@
+## The Hooks
+
+The library consists of three hooks that offer different interactions with your API. Each hook can be accessed by navigating through the controller and endpoint structure of your API client, using the result of the API Hooks `create` method as a starting point.
+
+For example, if you stored the result of the `create` method in a constant called `apiHooks`, like the above example, that constant can now be imported and used in any component, as long as that component is rendered anywhere within the `ApiHooksStore.Provider` component:
+
+```TypeScript
+import { apiHooks } from "*create method location*"
+
+const MyComponent: React.FunctionComponent = () => {
+
+  const [{data, isFetching}] = apiHooks.firstController.getEndpoint.useQuery();
```

Comment:
Very very minor and unimportant thing, but the formatting here is slightly different to elsewhere:

```ts
[{data, isFetching}]
```

vs

```ts
[{ data, isFetching }]
```